jmclient/jmclient/blockchaininterface.py

```diff
@@ -469,12 +488,28 @@ def core_proof_to_merkle_branch(self, core_proof):
         return core_proof[80:]
 
     def verify_tx_merkle_branch(self, txid, block_height, merkle_branch):
-        block_hash = self.rpc("getblockhash", [block_height])
-        core_proof = self.rpc("getblockheader", [block_hash, False]) + \
+        block_hash = self._rpc("getblockhash", [block_height])
+        core_proof = self._rpc("getblockheader", [block_hash, False]) + \
             binascii.hexlify(merkle_branch).decode()
-        ret = self.rpc("verifytxoutproof", [core_proof])
+        ret = self._rpc("verifytxoutproof", [core_proof])
         return len(ret) == 1 and ret[0] == txid
 
+    def listaddressgroupings(self):
+        return self._rpc('listaddressgroupings', [])
+
+    def listunspent(self, minconf=None):
+        listunspent_args = []
+        if 'listunspent_args' in jm_single().config.options('POLICY'):
+            listunspent_args = ast.literal_eval(jm_single().config.get(
+                'POLICY', 'listunspent_args'))
+        if minconf is not None:
```

Comment:
As I understand it, you added this parameter to support the usage currently in `BitcoinCoreNoHistoryInterface`. That makes sense, but why not maxconf also?

Also it's probably a good idea to add a docstring for this, e.g. here it could note that *if* set, the minconf parameter overrides the values in the config.

Reply:
> but why not maxconf also?

No reason, just nobody needs it currently.